parlai/agents/drqa/drqa.py

```diff
@@ -146,10 +146,6 @@ def _init_from_saved(self, fname):
         saved_params = torch.load(fname,
             map_location=lambda storage, loc: storage)
 
-        if 'word_dict' in saved_params:
-            # for compatibility with old saves
-            self.word_dict.copy_dict(saved_params['word_dict'])
-
```

Comment:
this section needs to stay

Reply:
was not deleted on purpose... thanks for catching!